apps/remix-ide/src/app/providers/injected-provider.tsx

```diff
@@ -61,8 +61,11 @@ export class InjectedProvider extends Plugin implements IProvider {
     }
     try {
       if ((window as any) && typeof (window as any).ethereum.request === "function") (window as any).ethereum.request({ method: "eth_requestAccounts" });
-      const resultData = await this.provider.currentProvider.send(data.method, data.params)
-      resolve({ jsonrpc: '2.0', result: resultData.result, id: data.id })
+      let resultData = await this.provider.currentProvider.send(data.method, data.params)
+      if (resultData && resultData.jsonrpc === '2.0') {
```

Comment:
shouldn't you also check resultData.jsonrpc before you check the value?
resultData && resultData.jsonrpc && resultData.jsonrpc === '2.0'

Reply:
ok sure, I don't think that change anything cause `jsonrpc` will be undefined then.